tests/unit/test_importer.py

```diff
@@ -48,3 +48,9 @@ def test_no_suppress_other_exception():
     with pytest.raises(Exception):
         with ImportExtensions(required=True, logger=default_logger):
             raise Exception
+
+
+def test_path_importer():
+    from jina.importer import PathImporter
+    with pytest.raises(ImportError):
+        PathImporter.add_modules('unit.error_package')
```

Comment:
The `ModuleNotFoundError` is not covered in this case.

Reply:
this error won't be handled by this function

Reply:
try this

```
with pytest.raises(FileNotFoundError):
    PathImporter.add_modules('some_package.py')
```